website/notifications/tasks.py

```diff
@@ -106,4 +105,4 @@ def remove_notifications(email_notification_ids=None):
     :return:
     """
     for email_id in email_notification_ids:
-        NotificationDigest.remove(Q('_id', 'eq', email_id))
+        NotificationDigest.objects.get(_id=email_id).delete()
```

Comment:
Minor: This loops can be replaced with `NotificationDigest.objects.filter(__id__in=email_notification_ids).delete()`.